jme3-core/src/main/java/com/jme3/material/RenderState.java

```diff
@@ -508,6 +512,10 @@ public void write(JmeExporter ex) throws IOException {
         oc.write(backStencilDepthPassOperation, "backStencilDepthPassOperation", StencilOperation.Keep);
         oc.write(frontStencilFunction, "frontStencilFunction", TestFunction.Always);
         oc.write(backStencilFunction, "backStencilFunction", TestFunction.Always);
+        oc.write(frontStencilReference,"frontStencilReference",0);
+        oc.write(backStencilReference,"backStencilReference",0);
+        oc.write(frontStencilMask,"frontStencilMask",Integer.MAX_VALUE);
+        oc.write(backStencilMask,"backStencilMask",Integer.MAX_VALUE);

```

Comment:
Please fix the formatting issue. (add space after ",")

Reply:
done

Reply:
thanks